### DISCUSSION TITLE
Merge Background source and Seasonal backgrounds options

### DISCUSSION THREAD

**Discussion by Nekomosh004** [ppy/osu#20352] [Ideas]
Suggestion: Merge Background source and Seasonal backgrounds options

Currently it shows both separated.
![image](https://user-images.githubusercontent.com/82471957/190860295-c8b297ac-f24d-40eb-8a27-f4aff326072a.png)

Instead of having these two options separated, I suggest merging them, showing only the following:

Skin <sup>1</sup>
Beatmap <sup>2</sup>
Seasonal background <sup>3</sup>
Beatmap or Seasonal background <sup>4</sup>
Beatmap (with storyboard/video) _(supporter only)_ <sup>5</sup>
Beatmap (with storyboard/video) or Seasonal background _(supporter only)_ <sup>6</sup>

<sup>1</sup> Always use the skin's background.
<sup>2</sup> Always use the playing beatmap's background.
<sup>3</sup> _**New.**_ Replacement for the "Always" option for the Seasonal backgrounds dropdown.
<sup>4</sup> _**New.**_ Replacement for the "Sometimes" option for the Seasonal backgrounds dropdown.
<sup>5</sup> If the beatmap has no storyboard/video, Beatmap background will be used.
<sup>6</sup> _**New.**_ If the beatmap has no storyboard/video, then Seasonal background will be used.

I'm suggesting this to simplify it, instead of separating them. The options menu has a lot of options right now, its scary 😨 
Thank you for reading c:

**Comment by WindowsMeosu** [comment]
seasonal background(s) being used on maps without a background/storyboard/video (or if storyboard/video is disabled) seems like a good idea, however I would also suggest adding something to specify that seasonal backgrounds are currently being used since following beatmap doesn't have a background/storyboard/video. I've proposed a suggestion for this matter below.

suggestion - Song Selection: There should be a text below the metadata (tags) that should read "This beatmap does not contain any background, storyboard, or video. Seasonal backgrounds may be used if you currently have them enabled." or something related.
